.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: deploy
```

Comment:
This file will be temporary before being combined into the one deploy.yml. For now, we need to give it a different name, say `dev deploy`,  otherwise you wont be able to tell which is which from the action tab.

Reply:
done